manifests/cluster-authentication-operator_06_clusteroperator.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  name: openshift-authentication
+spec: {}
```

Comment:
omit all together ?

Reply:
I shamelessly copied this from kube-apiserver-operator, let's see what happens when we remove it.